### ISSUE TITLE
Update examples to use ecosystem specific images 

### ISSUE THREAD

**Issue by yeikel** [dependabot/dependabot-script#900]
Now that the `dependabot/dependabot-core`  docker image is no longer updated as we are [building ecosystem-specific images ](https://github.com/orgs/dependabot/packages?repo_name=dependabot-core) the examples need to be updated to reflect that

**Comment by jeffwidman** [comment]
Yep, we know we need to do something here, but there's some design decisions so we aren't quite clear on the path forward yet.

In the meantime I put this up to provide a heads up:
* https://github.com/dependabot/dependabot-script/pull/906

**Comment by sheim-dev** [comment]
Is there any progress on this yet?

If there's any documentation or at least some hints available on how to switch to the ecosystem-specific images, I would be willing to help out with adapting the dependabot-script repo to those.

**Comment by jeffwidman** [comment]
We've updated this repo's README with a [deprecation note](https://github.com/dependabot/dependabot-script?tab=readme-ov-file#this-repo-is-deprecated) which points to a [new example repo](https://github.com/dependabot/example-cli-usage/) we setup to demonstrate how to self-host Dependabot using the [Dependabot CLI](https://github.com/dependabot/cli). As a result, we are archiving this repo.
